cpu/native/async_read.c

```diff
@@ -97,6 +97,34 @@ void native_async_read_add_handler(int fd, void *arg, native_async_read_callback
     _next_index++;
 }
 
+void native_async_read_remove_handler(int fd)
+{
+    int res = real_fcntl(fd, F_GETFL);
+    if (res < 0) {
+        err(EXIT_FAILURE, "native_async_read_remove_handler(): fcntl(F_GETFL)");
+    }
+    unsigned flags = (unsigned)res & !O_ASYNC;
```

Comment:
Should that be `~O_ASYNC`?

Reply:
@derMihai and @benpicco ⬆️

Reply:
Ouch, yes, `~O_ASYNC`. Why was this merged?

Reply:
Haha ok I'll fix is

Reply:
Don't worry, shit happens 😉